examples/4337-gas-metering/README.md

```diff
@@ -10,14 +10,14 @@ NOTE: If you run a paymaster analysis twice or more without changing the salt fo
+| **Alchemy (ETH - Gas Policy)**                                   | Yes            | 411372 ([TX](https://sepolia.etherscan.io/tx/0xcbb2c3c49b9d72d9ecf692308d69a8ad797ab5b1c6603f4fad989f966d692af1)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0xcbb2c3c49b9d72d9ecf692308d69a8ad797ab5b1c6603f4fad989f966d692af1/gas-usage))      | 418779 ([TX](https://sepolia.etherscan.io/tx/0x49fbedf833cfecf9db7de56c61d4227292723115520600dbc3711da5e6a85672)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0x49fbedf833cfecf9db7de56c61d4227292723115520600dbc3711da5e6a85672/gas-usage))      | 130202 ([TX](https://sepolia.etherscan.io/tx/0x35f1e5b04d988e4614a17609190b3e21b0a9892f78da9f400248cfb3b5afde9a)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0x35f1e5b04d988e4614a17609190b3e21b0a9892f78da9f400248cfb3b5afde9a/gas-usage))     | 421926 ([TX](https://sepolia.etherscan.io/tx/0x7dda913ae986d49c4322f414102ae374441a40adb4b33727e568ba140904d52a)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0x7dda913ae986d49c4322f414102ae374441a40adb4b33727e568ba140904d52a/gas-usage))      | 133394 ([TX](https://sepolia.etherscan.io/tx/0xe34902ebd5377cac04c47d142f6ca2de558df63a7e0c6541f704df651b7cfcb1)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0xe34902ebd5377cac04c47d142f6ca2de558df63a7e0c6541f704df651b7cfcb1/gas-usage))     | 451200 ([TX](https://sepolia.etherscan.io/tx/0xb1253508bc4ca5ce41222b15b0e7bf03b2273bcb09d93e1d6d6a5ea39b43ee84)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0xb1253508bc4ca5ce41222b15b0e7bf03b2273bcb09d93e1d6d6a5ea39b43ee84/gas-usage))      | 162654 ([TX](https://sepolia.etherscan.io/tx/0xd13fb70626a26aaa02e0389cd9347c1c0d8d8ed9ee794a61c5d3eea4b36e239a)) ([Gas](https://dashboard.tenderly.co/tx/sepolia/0xd13fb70626a26aaa02e0389cd9347c1c0d8d8ed9ee794a61c5d3eea4b36e239a/gas-usage))      |
```

Comment:
Fair enough, I'll remove them and update the summary table once I return next week. Or if you have capacity, I'd appreciate your help 👍

Reply:
Ideally, we would be able to do this programmatically instead of requiring multiple updates. Not in scope for this sprint, but some potential tech debt.

Reply:
Not sure if tenderly supports that, I have asked them in the channel.

Reply:
You should be able to do this without Tenderly.

Reply:
New question related to the summary table @remedcu @nlordell:
> The table uses the bundler’s ethereum transaction gas instead of the user operation gas used. Would it make sense to use the latter? **Reason**: To compare this number across the available bundlers, we’d require all bundlers to include only a single user op in a transaction, which can’t be guaranteed. Also, if the bundler decides to execute arbitrary logic (which affects the number, but we’re only paying for the user op) this would affect the comparison

Another example that highlights that transaction gas limit might be not what you're paying to the bundler:
> ERC721 Minting User Op with Pimlico Verifying Paymaster:
Gas Used (Account or Paymaster): 170150
Gas Used (Transaction): 152766
The first number is the user operation, the second is the actual ethereum transaction

Reply:
With ERC20 Paymaster, using the user operation gas we paid for the user op makes sense. But with the gas policy one, using the former might be better as we are not paying the gas for that (at least directly in some cases, where the policy is sponsored by say a protocol or company) to make the sponsors aware if extra arbitrary logic is being executed with the transaction and which bundler's gas policy might be the right approach.